ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -127,6 +138,24 @@ export default function CustomSpendingCap({
     }
 
     dispatch(setCustomTokenAmount(String(valueInput)));
+
+    const newData = getCustomTxParamsData(txParams.data, {
```

Comment:
Description from the related issue ticket:
> We likely only need to recalculate it once user lands in the second screen (where we display the gas info).

was this considered? wondering why we recalculate on this page rather than only calculating on the second screen